v0/destinations/google_adwords_enhanced_conversions/transform.js

```diff
@@ -10,6 +12,22 @@ const ErrorBuilder = require("../../util/error");
 
 const { trackMapping, BASE_ENDPOINT } = require("./config");
 
+/**
+ * This function is helping to update the mappingJson.
+ * It is removing the metadata field with type "hashToSha256"
+ * @param {} mapping -> it is the configMapping.json
+ * @returns
+ */
+const updateMappingJson = mapping => {
+  const newMapping = [];
+  mapping.forEach(element => {
+    if (get(element, "metadata.type"))
+      if (element.metadata.type === "hashToSha256") delete element.metadata;
```

Comment:
Instead of deleting entire `metadata` from mappingJson it is better to set `type` to `toString`

Reply:
If that is a feature in construct payload lets do that `delete` is a risky key word